general/g.version/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+import os
+from types import SimpleNamespace
+import pytest
+import grass.script as gs
+
+
+@pytest.fixture(scope="module")
+def grass_session(tmp_path_factory):
+    """Set up a GRASS session for the tests."""
+    tmp_path = tmp_path_factory.mktemp("grass_session")
+    location = "test_location"
+    gs.core._create_location_xy(tmp_path, location)  # pylint: disable=protected-access
```

Comment:
[gs.create_project()](https://grass.osgeo.org/grass84/manuals/libpython/grass.script.html#grass.script.core.create_project) will now work in this context with both `tmp_path / project` or `tmp_path, project` without `disable=protected-access`.

Reply:
Done